src/components/Product/Product.js

```diff
@@ -26,6 +26,7 @@ function Product() {
       .onSnapshot((snapshot) => {
         let products = [];
         snapshot.forEach((doc) => products.push(format(doc.data().name)));
+        console.log(products);
```

Comment:
Some debugging happening here, do you guys need this line in production? :trollface:

Reply:
jajajajajajaaj ups!!  Thank you for notifying us about this.

Reply:
OMG I didn't know :trollface: was a thing! LOL

But seriously, you need to delete that `console.log` :P